snarkos/ledger/server.rs

```diff
@@ -198,12 +198,21 @@ impl<N: Network> Server<N> {
 
         // POST /testnet3/program/deploy
         let deploy_program = warp::post()
-            .and(warp::path!("testnet3" / "deploy"))
+            .and(warp::path!("testnet3" / "program" / "deploy"))
+            .and(warp::body::content_length_limit(10 * 1024 * 1024))
```

Comment:
do we know the size limit of programs? that would be the best fit here and in `/testnet3/program/execute`

Reply:
Good question, I actually haven't thought about this, just used the same number as the transaction broadcast and deploy endpoint

Reply:
the same applies to the other types of data that the protocol constrains, including the transaction :+1:

Reply:
@raychu86 @howardwu any input on those body lengths?